create HA doc

bybai opened the pull request:
for https://github.ibm.com/vhu/c910env/issues/304 and https://github.com/xcat2/xcat2-task-management/issues/160

UT link: http://10.5.106.1/install/coral_ha/xcat-core/docs/build/html/references/coral/ha/index.html

whowutwut commented on docs/source/references/coral/ha/ha_mgmt_node_with_shared_data.rst:
```````````````````````````````````````````````````````````````````diff
@@ -163,114 +178,12 @@ Use ``xcatha.py -a`` to start all related services: ::
-#. Provision an existed compute node like ``cn1`` successfully.
+Unplanned failover: primary xCAT management node is not accessible
+``````````````````````````````````````````````````````````````````
+If primary xCAT management node becomes not accessible before being deactivated and backup xCAT management node is activated, it is recommended that the primary node is disconnected from the network before being rebooted. This will ensure that when services are started on reboot, they do not interfere with the same services running on the backup xCAT management node.
```````````````````````````````````````````````````````````````````
@bybai @gurevichmark  The whole purpose of this exercise is to find out and disable the services so they **do not** start up at boot.   Writing this snip means we do not know 100% that we have covered everything and do not know what will happen upon reboot of the old MN node in HA, we need to re-visit this.

whowutwut commented on docs/source/references/coral/ha/ha_mgmt_node_with_shared_data.rst:
```diff
@@ -163,114 +178,12 @@ Use ``xcatha.py -a`` to start all related services: ::
     2018-06-24 22:13:19,353 - DEBUG - makeconservercf [Passed]
     2018-06-24 22:13:19,449 - DEBUG - systemctl start conserver [Passed]
```
Should these messages  contain `DEBUG` ?

bybai commented:
Hi @gurevichmark,
I modified some steps. 
1. The first part is how to configure primary HA MN, so I added the section title. After it is configured, admin can use this MN, so I added activate step.
2. The second part is how to configure and activate standby HA MN. In this part, user can use an existed MN or install a new standby MN, so install xcat without VIP on ``xcatmn2`` local disk and switch DB are first steps, and these steps are more flexible, the standby node can be installed any time and no effect to active primary MN node . When we want activate standby MN, it can be configured as HA MN using VIP and shared data, and then activate.
